app-center-services/src/main/java/org/exoplatform/appcenter/service/ApplicationCenterService.java

```diff
@@ -687,7 +745,7 @@ private boolean hasPermission(String username, String permissionExpression) {
       return false;
     }
 
-    if (StringUtils.equals(IdentityConstants.ANY, permissionExpression)) {
+    if (StringUtils.equals(DEFAULT_USERS_PERMISSION, permissionExpression)) {
```

Comment:
Does this is mandatory ? as it is a classic permission, it will be check by the next case

Reply:
Yes since we updated the default permissions to (*:/platform/users) and to avoid the rest of the checks.

Reply:
You are right i will remove it.